helm-chart/scc-admin-frontend/templates/deploy-secret-serviceaccount.yaml

```
@@ -0,0 +1,34 @@
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "10"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: create-secrets
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "10"
+rules:
+  - apiGroups: [""] # "" indicates the core API group
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)